### PR TITLE
Branch three

### PR DESCRIPTION
**Pull request by yashnevatia**

### CODE REVIEW

**Comment by yashnevatia** [comment]
Screenshot 2021-10-03 at 7 01 43 PM

File: bibblio/lambda_folder/pipetest.py [hunk 1]
```diff
@@ -0,0 +1,181 @@
+
```
Comment (vc2309) [inline bibblio/lambda_folder/pipetest.py:1]: Why do we need this? We can use `sam invoke` to do a local test

File: bibblio/tests/unit/test_snapshots_dao.py [hunk 2]
```diff
@@ -30,5 +30,7 @@ def test_get_snapshots_empty(snapshots_dao):
     assert snapshots_dao.get_snapshots_by_user_id("...") == []
 
 
-# def test_write_snapshot(snapshots_dao):
-#     snapshot_id = snapshots_dao.
+def test_update_snap_status(insert_snapshots, snapshots_dao):
+    snapshots_dao.update_snap_status("example-snapshot-01", "Delivered")
+    snap = snapshots_dao.get_item_by_id("example-snapshot-01")
+    snap["delivery_status"] == "Delivered"
```
Comment (vc2309) [inline bibblio/tests/unit/test_snapshots_dao.py]:
```suggestion
    assert snap["delivery_status"] == "Delivered"
```

File: bibblio/lambda_folder/app.py [hunk 3]
```diff
@@ -57,13 +59,17 @@ def test_handler(event, context):
...
+    for snap in delivery_snaps:
+        content = snap_engine.get_snap_content_by_snap_id(snap['snap_shot_id'],
          snap_shot_dao, SmartNotesDAO())
+        email_id = user_dao.get_user_by_userid(snap['user_id'])['email_id']
         email_engine.send_email(email_id, content)
+        snap_shot_dao.update_snap_status(snap['snap_shot_id'], 'Delivered')
+        # for smart_note_id in snap["smart_note_list"]:
+        #     smart_note_dao.update_smart_note_status(smart_note_id, snap['snap_shot_id'])
+        # TODO: update status of snap
+        # TODO: update status of smart notes
```
Comment (vc2309) [inline bibblio/lambda_folder/app.py]: These to dos will be done here or in fututrE?

**Review by vc2309 — APPROVED** [review]

**Pull request merged**